### PR TITLE
fixed flow name

### PR DESCRIPTION
**Pull request by edewit**
fixes: #1045

### CODE REVIEW

File: src/authentication/DuplicateFlowModal.tsx [hunk 1]
```diff
@@ -46,12 +46,12 @@ export const DuplicateFlowModal = ({
...
       await adminClient.authenticationManagement.copyFlow({
         flow: name,
-        newName: form.name,
+        newName: form.alias,
       });
       if (form.description !== description) {
         const newFlow = (
           await adminClient.authenticationManagement.getFlows()
-        ).find((flow) => flow.alias === form.name)!;
+        ).find((flow) => flow.alias === form.alias)!;
```
Comment (jonkoops) [inline src/authentication/DuplicateFlowModal.tsx:54]: We should not be using the non-null assertion operator here as the result from a `find()` operation could possibly be `undefined`.
```suggestion
        ).find((flow) => flow.alias === form.alias);
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: src/authentication/DuplicateFlowModal.tsx (hunk 1 above)
Comment (edewit) [reply src/authentication/DuplicateFlowModal.tsx:54]: how can it not be found as it's just been created 1 line above

Comment (jonkoops) [reply src/authentication/DuplicateFlowModal.tsx:54]: Of course, but there are no guarantees.

**Review by jonkoops — APPROVED** [review]
Looks good enough :+1:

**Review by ssilvert — APPROVED** [review]
Tested and it's working now.  Approved.

**Pull request merged**